pkg/templateservicebroker/openservicebroker/api/types.go

```diff
@@ -59,6 +59,14 @@ type ServiceBindings struct {
 	Create map[string]*jsschema.Schema `json:"create,omitempty"`
 }
 
+type ParameterSchemas struct {
+	ServiceInstance ParameterSchema `json:"service_instance,omitempty"`
+}
+
+type ParameterSchema struct {
+	Create map[string]interface{} `json:"create,omitempty"`
```

Comment:
you don't want to make Create of type struct {
OpenShiftFormDefinition []string `json:"openshift_form_definition,omitempty"`
}
?

Reply:
I guess that would make sense :)

Reply:
updated.